README.md

```diff
@@ -222,6 +259,28 @@ You'll be redirected automatically to an Okta login page. Once you successfully
 
 This module integrates with Spring Security's OAuth support, all you need is the mark your application with the standard `@EnableOAuth2Client` annotation. 
 
+## Proxy
+
+If you're running your application (with this okta-spring-boot dependency) from behind a network proxy, you could add JVM args to your application like:
+
+```bash
+-Dhttps.proxyHost=host
+-Dhttps.proxyPort=port
+-Dhttps.proxyUser="user"          # optional
+-Dhttps.proxyPassword="password". # optional 
+```
+
+or, you could set it programmatically like:
+
+```java
+System.setProperty("https.proxyHost", "https://example-proxy.com");
+System.setProperty("https.proxyPort", "443");
+```
+
+See [here](https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html) for the complete list of properties.
+
+**Note:**  Spring WebFlux (and `WebClient`) do not support these properties. (See [spring-projects/spring-security#8882](https://github.com/spring-projects/spring-security/issues/8882)).
+
```

Comment:
Can you move these out to another PR, we can probably merge this separately.
Just make sure you tested against a proxy before merging into the readme (NOTE: IIRC `localhost` is in the default non-proxy-host list)

Reply:
Yes, I merged the master into this feature branch and that is why these changes show up here. Will revert that. Once this PR is merged, I'll create a separate PR to merge `release_v2` into master.

Reply:
Done